sdl2promela/sdl/model.py

```diff
@@ -972,12 +986,26 @@ def __gather_states(self):
             state.name = stateName
             self.states[stateName] = state
 
+    def __gather_continuous_signals(self):
+        for state in self.states:
+            self.continuous_signals[state] = []
+        for state, signals in self.source.cs_mapping.items():
+            for signal in signals:
+                cs = ContinuousSignal()
+                cs.trigger = convert(signal.trigger.question)
+                cs.transition = signal.transition_id
+                self.continuous_signals[state].append(cs)
+
     def __get_inputs_of_name(self, name: str) -> List[ogAST.Input]:
         result = set()
         for inputs in self.source.mapping.values():
             if isinstance(inputs, List):
                 for input in inputs:
-                    if isinstance(input, ogAST.Input) and (name in input.inputlist):
+                    if (
+                        isinstance(input, ogAST.Input)
+                        and (name in input.inputlist)
+                        and len(input.inputlist) == 1
```

Comment:
Why == 1?

Reply:
To get proper number of signal parameters: The `*` or `StateA,StateB,StateX` cannot be used to get valid parameter list.

Reply:
OK, so I think this needs a comment and possibly TODO

Reply:
Added: see last commit